tests/unistd/dup.c

```diff
@@ -12,14 +12,41 @@
 
 int main() {
   int f, f2, f3;
+  off_t offset;
+
+  char *fnam = "/tmp/emscripten_test_file";
+  remove(fnam);
+  errno = 0;
+
+  f = open(fnam, O_RDWR | O_CREAT, S_IRUSR | S_IRGRP | S_IROTH);
+  close(f);
 
   printf("DUP\n");
-  f = open("/", O_RDONLY);
-  f2 = open("/", O_RDONLY);
+  f = open(fnam, O_RDONLY);
```

Comment:
I would rather that we not modify tests like this, as there is a risk of losing existing coverage (maybe this test cared about `/` specifically? I'm not sure).

Adding new test case at the end of the existing tests seems safest to me.

Reply:
Done, let me know if this approach works better.